netutils/interface.py

```diff
@@ -345,15 +359,15 @@ def _split_interface_tuple(interface: str) -> t.Tuple[CharacterClass, ...]:
                 part += interface[idx]
                 idx += 1
             if part and idx == len(interface):
-                tail = (*tail, cls(part, True))
+                tail = (*tail, cls(part, True))  # type: ignore
                 break
             if part:
-                tail = (*tail, cls(part))
+                tail = (*tail, cls(part))  # type: ignore
```

Comment:
There probably is a way to resolve these, but I deem it not worth it for now. Something like `Union[Callable[[str, Optional[bool]], CharacterClass], Callable[[...], NoReturn]` might do the trick if you want to play around with it a little.